### ISSUE TITLE
fix: keyboard navigation does not work in FluentTabs

### ISSUE THREAD

**Issue by jloeffelholz** [microsoft/fluentui-blazor#4158]
# 🐛 Bug Report

When navigating through tabs with the keyboard, keyboard navigation should go to inactive tabs and allow selection with the keyboard. Instead, only the active tab can be selected with keyboard navigation and inactive tabs are not accessible by keyboard navigation.

## 💻 Repro or Code Sample
This is reproducible on the demo site: https://www.fluentui-blazor.net/Tabs

## 🤔 Expected Behavior

Keyboard navigation should allow the user to navigate to inactive tabs via the tab key and use the enter key to make that tab active.

## 💁 Possible Solution

Possibly setting the tab index on inactive tabs to 0 instead of -1 and ensuring that enter keys on inactive tabs activates the tab.

## 🔦 Context

This issue is causing our site to not pass accessibility tests due to tab navigation not working correctly.

**Comment by MarvinKlein1508** [comment]
@jloeffelholz you can navigate the tabe using the arrow keys. With tab you can only select the control bar for the tabs itself. It's the same behaviour for v5 https://fluentui-blazor-v5.azurewebsites.net/Tabs

@vnbaaij do we want to add support for tab here as well? Or is there any reason we only support arrow keys?

**Comment by vnbaaij** [comment]
The underling web component only supports arrow keys so we cannot (easily, if possible at all) add tab support there. Closing this as a11y support is available with the arrow keys.
